st2actions/st2actions/runners/ssh/paramiko_ssh.py

```diff
@@ -351,6 +351,10 @@ def run(self, cmd, timeout=None, quote=False):
         chan = transport.open_session()
 
         start_time = time.time()
+        if cmd.startswith('sudo'):
```

Comment:
A bit late, but `cmd.startswith('sudo ')` is probably a bit safer (what if someone has a binary which name starts with sudo) :P